modules/basis/basis-datasource/smart-datasource/pom.xml

```diff
@@ -0,0 +1,40 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.sap.xsk</groupId>
+  <artifactId>smart-datasource</artifactId>
```

Comment:
Can this be part of  `modules/basis/basis-datasource/` instead of  `modules/basis/basis-datasource/smart-datasource/`? Maybe the `smart-datasource` folder is not really needed?